package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "ng2-translate",
   "description": "An implementation of angular translate for Angular 2",
+  "private": true,
   "scripts": {
     "test": "karma start",
     "test-watch": "karma start --singleRun=false --autoWatch=true",
-    "commit": "npm run prepublish && npm test && git-cz",
-    "prepublish": "ngc && npm run build",
-    "build": "webpack && cp bundles/ng2-translate.umd.js bundles/index.js",
+    "commit": "run-s prepublish test git-cz",
```

Comment:
this is usage of `npm-run-all` nice cross platform way to run npm scripts in series or parallel
I had a question about `git-cz` how to use it